yt_dlp/YoutubeDL.py

```diff
@@ -4070,7 +4073,7 @@ def _write_subtitles(self, info_dict, filename):
         return ret
 
     def _write_thumbnails(self, label, info_dict, filename, thumb_filename_base=None):
-        ''' Write thumbnails to file and return list of (thumb_filename, final_thumb_filename) '''
+        ''' Write thumbnails to file and return list of (thumb_filename, final_thumb_filename) or None if error '''
```

Comment:
just to keep consistent with the `_write_subtitles` docstring
```suggestion
        ''' Write thumbnails to file and return list of (thumb_filename, final_thumb_filename); or None if error '''
```